scripts/codemods/explicit-barrel-imports.cjs

```diff
@@ -1,39 +1,43 @@
-const fs = require('fs');
 const path = require('path');
 
+const isBareSpecifier = (importPath) => !importPath.startsWith('app/') && /^[^./]/.test(importPath);
+const barrelFileNames = ['index.ts', 'index.tsx', 'index.js', 'index.jsx'];
+
+const resolvePath = (fileDir, importPath) => {
+  if (importPath.startsWith('app/')) {
+    const resolvedPath = require.resolve(path.join(process.cwd(), 'public', importPath));
```

Comment:
**Nit:** in case someone wouldn't run this from the root of the repo, maybe something like this would be more resilient?
```js
const resolvedPath = require.resolve(path.join(__dirname, '..', '..', 'public', importPath));
```